lib/MusicBrainz/Server/Data/WebService.pm

```diff
@@ -201,11 +201,11 @@ sub xml_search
     }
 
     $query =~ s/^ AND //;
-    # In case we have a blank query
-    if ($query =~ /^\s*$/)
+    # In case we have a blank query (only whitespace, or whitespace surrounded by unescaped quotes)
+    if ($query =~ /^['"]?\s*['"]?$/)
     {
         return {
-            error => "Must specify a least one parameter (other than 'limit', 'offset' or empty 'query') for collections query.",
+            error => "You submitted a blank query. You must include a non-blank 'query' parameter with your query.",
```

Comment:
Nit: the word query at the end sounds a bit funny/repetitive with what comes before. Maybe "with your search" instead?